CHANGELOG.md

```diff
@@ -28,6 +28,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
  - Fixed AXI-inval-filter policy for D$ lines invalidation upon vector stores that are misaligned w.r.t. the D$ line width
  - Fix lane sequencer checks for floating-point comparisons
  - Fix synthesis error occuring due to the continuous assignmnet in the always block of mask unit
+ - Fix wrong variable in `vmerge` and `vmv` `riscv-tests`
```

Comment:
That is not part of this PR

Reply:
It can be considered part of the update since they fixed an LLVM bug and made `spike` crash without the fix. Without this, the CI won't pass ;-)